### ISSUE TITLE
Conversations and edits

### ISSUE THREAD

**Issue by kiil** [pachecoio/openai_cli#1]
It seems, every prompt starts over.

Would you add conversations to the text generation?

And perhaps edit to the image generation?

**Comment by sudoish** [comment]
Yes, I intend to add those features too,  the CLI is very basic at the moment.
